doc/interactors.md

```diff
@@ -182,7 +183,7 @@ The checkbox element
 
 ##### Synopsis
 
-```javascript
+````javascript
```

Comment:
I think we only want three ` characters

Reply:
Autoformat goofing things up compensating for another errant backtick. Also thought I had that off, but I turned it off in the wrong settings file.